packages/shared/lib/clients/orchestrator.ts

```diff
@@ -389,22 +425,31 @@ export class Orchestrator {
                 groupKey,
                 args
             });
-            const res = result.mapError((e) => new NangoError('post_connection_failure', e.payload ?? { error: e.message }));
+
+            const res = result.mapError((err) => {
+                return (
+                    deserializeNangoError(err.payload) ||
+                    new NangoError('post_connection_script_failure', { error: err.message, ...(err.payload ? { payload: err.payload } : {}) })
+                );
+            });
 
             if (res.isErr()) {
                 throw res.error;
             }
 
-            const content = `The post connection script workflow ${workflowId} was successfully run. A truncated response is: ${JSON.stringify(res.value, null, 2)?.slice(0, 100)}`;
+            const content = `The post connection script was successfully run.`;
 
-            await logCtx.info(content);
+            await logCtx.info(content, {
+                postConnection: name,
+                connection: connection.connection_id,
+                integration: connection.provider_config_key
+            });
```

Comment:
res.value is no longer logged?

Reply:
since post connection scripts are not inline they are not supposed to have meaningful output so I removed the logging of the result. Let me know if you disagree